File: 1_beginner/2783.py
# https://www.urionlinejudge.com.br/judge/en/problems/view/2783

N, C, M = input().split(' ')
stamped = [int(x) for x in input().split(' ')]
purchased = [int(x) for x in input().split(' ')]

print(len([x for x in stamped if x not in purchased]))

File: 1_beginner/2783.c
// https://www.urionlinejudge.com.br/judge/en/problems/view/2783

#include <stdio.h>
#include <string.h>

int main()
{
    int N, C, M;
    int stamped[50], purchased[101];
    int i, p, missing;

    memset(purchased, 0, sizeof(purchased));

    scanf("%d%d%d", &N, &C, &M);
    for (i = 0; i < C; ++i) {
        scanf("%d", &stamped[i]);
    }
    for (i = 0; i < M; ++i) {
        scanf("%d", &p);
        purchased[p] = 1;
    }

    missing = C;
    for (i = 0; i < C; ++i) {
        if (purchased[stamped[i]])
            --missing;
    }

    printf("%d\n", missing);

    return 0;
}